README.md

```diff
@@ -170,12 +171,21 @@ Immediately triggers a new search for the given text
 
 - text `String` A word of sequence of words to search for
 
-#### `search.playResult(result)`
+**Returns:**
+- retVal `Promise` - A promise that will resolve with the results of the search you just performed.
+The promise will be rejected with a timeout error if the search takes too long.
+
+#### `search.playResult(searchText, result)`
 Immediately attempts to play the given result.  If we fail to play the given result
 an error will be thrown.
 
+- searchText `String` The search text that causes the search result to appear
```

Comment:
You are right about it being strange but it's also strange that we can only
play search results and not an album directly.

I already explained a scenario when searching isn't necessary; if the user
does navigation in the UI on their own
On Jun 19, 2016 18:16, "Samuel Attard" <notifications@github.com> wrote:

> In README.md
> <https://github.com/gmusic-utils/gmusic-ui.js/pull/3#discussion_r67619463>
> :
>
> >  Immediately attempts to play the given result.  If we fail to play the given result
> >  an error will be thrown.
> >
> > +- searchText `String` The search text that causes the search result to appear
>
> Quite possibly, but as a user I would still find it strange that I can't
> playResult() without first calling performSearch(). There is no case
> where a user calling playResult() can get away with not calling
> performSearch() directly before it.
>
> In it's previous state and your desired state playResult() will be a
> useless function by itself. I don't see how crippling a function like that
> is a desired effect
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gmusic-utils/gmusic-ui.js/pull/3/files/9140981c7fed0e92b2a9c1ed58c5b86e4d00dcba..0910e48755e7a5c1f9d5085c45599c602f52324c#r67619463>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA3FWI8RgUA-du1nV12fwk1oU7C6lor6ks5qNWtcgaJpZM4I462I>
> .
>